### ISSUE TITLE
occur with "minibuffer-inactive ... ..."

### ISSUE THREAD

**Issue by fu123456** [benma/visual-regexp.el#59]
I use the package awesome-tray package https://github.com/manateelazycat/awesome-tray, but I found that I use vr/replace, and my emacs output some undesirable information as follows:
![DeepinScreenshot_emacs_20200930104507](https://user-images.githubusercontent.com/22411577/94637921-4f023380-030b-11eb-9e0f-27181d29cafc.png)

So, I want to know how to prohibit this message which is often nosiy. Thanks.

**Comment by benma** [comment]
I have never seen anything like it. Please try asking the same at the awesome-tray issue tracker.

**Comment by OldhamMade** [comment]
I'm also seeing something similar:

<img width="895" alt="Screen Shot 2021-01-25 at 10 32 42" src="https://user-images.githubusercontent.com/111193/105687378-ab134800-5ef8-11eb-9659-c1d7cf95b7b6.png">

This is on Emacs 27, specifically `d12frosted/emacs-plus/emacs-plus@27` from homebrew:

<img width="895" alt="Screen Shot 2021-01-25 at 10 33 27" src="https://user-images.githubusercontent.com/111193/105687468-c4b48f80-5ef8-11eb-8171-87b906d5f4da.png">

It's the same with Emacs 28 also.

I don't use awesome-tray, so I don't think it's that. I'm also seeing that the initial search doesn't find/highlight anything:

<img width="779" alt="Screen Shot 2021-01-25 at 10 35 53" src="https://user-images.githubusercontent.com/111193/105687752-1d842800-5ef9-11eb-8f6c-ac98935c3ac3.png">

This feels more like an issue with 27+ rather than any specific package interaction.

**Comment by OldhamMade** [comment]
Ah, interesting: there seems to be some odd interaction between this package and [feebleline](https://github.com/tautologyclub/feebleline). If I disable that package, everything works fine.

**Comment by p1llule** [comment]
Indeed. both feebleline, awesome-tray, taoline and visual-regexp use extensively the echo area and so can conflict.

[My workaround is to put a new hook in visual-regexp PR](https://github.com/benma/visual-regexp.el/pull/60)
Then to add some custom functions to neutralize (deactivating it without reactivating the modeline) the other package while using visual-regexp :

`  (defun pill--vr/initialize-with-taoline ()
    "Disabilitates `tao-line'."
    (when taoline-mode
      (setq pill/taoline t)
      (cancel-timer taoline/timer)
      (remove-hook 'focus-in-hook 'taoline-mode-line-proxy-fn)))
  (add-hook 'vr/initialize-hook #'pill--vr/initialize-with-taoline)

  (defun pill--vr/end-with-taoline ()
    "Reabilitates `tao-line'."
    (when pill/taoline
      (setq taoline/timer
            (run-with-timer 0 0.5 'taoline-mode-line-proxy-fn))
      (add-hook 'focus-in-hook 'taoline-mode-line-proxy-fn)))
  (add-hook 'vr/end-hook #'pill--vr/end-with-taoline)`
